src/detect_breakends.cpp

```diff
@@ -23,14 +23,32 @@ auto enumeration_names(clustering_methods)
                                                   {"3", clustering_methods::candidate_selection_based_on_voting},
                                                   {"candidate_selection_based_on_voting",
                                                    clustering_methods::candidate_selection_based_on_voting}};
-}
+};
+
+// Specialise a mapping from an identifying string to the respective value of your type refinement_methods. With the
+// help of this function, you're able to call ./detect_breackends with -r 0 and -r no_refinement and get the same
+// result.
+auto enumeration_names(refinement_methods)
+{
+    return std::unordered_map<std::string_view,
+                              refinement_methods>{{"0", refinement_methods::no_refinement},
+                                                  {"no_refinement",
+                                                   refinement_methods::no_refinement},
+                                                  {"1", refinement_methods::sViper},
+                                                  {"sViper",
+                                                   refinement_methods::sViper},
+                                                  {"2", refinement_methods::prioritzation_based_on_rank_aggregation},
+                                                  {"prioritzation_based_on_rank_aggregation",
+                                                   refinement_methods::prioritzation_based_on_rank_aggregation}};
+};
 
 struct cmd_arguments
 {
     std::filesystem::path alignment_file_path{};
     std::filesystem::path insertion_file_path{};
     std::vector<uint8_t> methods{1, 2, 3, 4};                   // default is using all methods
     clustering_methods clustering_method{simple_clustering};    // default is the simple clustering method
+    refinement_methods refinement_method{no_refinement};        // default is using no refinement
```

Comment:
Why not make the default to using sViper?

Reply:
Also we have to consider that sViper will only work if the user provides long reads AND short reads. With only one dataset given, the command parser should probably throw an error if sViper refinement is selected.

Reply:
Yes, we have to solve https://github.com/seqan/igenvar/issues/43 next, and there we could add a check if sViper is possible to use when we know which input we have.